lib/readyset/configuration.rb

```diff
@@ -2,38 +2,36 @@
 
 module Readyset
   class Configuration
-    attr_accessor :connection_url, :database_selector, :database_resolver,
-      :database_resolver_context
+    attr_accessor :database_url
 
     def initialize
-      @connection_url = ENV['READYSET_URL'] || default_connection_url
-      @database_selector = { delay: 2.seconds }
-      @database_resolver = Readyset::DefaultResolver
-      @database_resolver_context = nil
+      @database_url = ENV['READYSET_URL'] || default_connection_url
```

Comment:
I spent some time looking into #36, and I decided that it wasn't worth the trouble for the time being, which means I don't think we'll need to take the readyset connection string as a config option (since the connection info will be defined in `config/database.yml`

Reply:
> It seems like that's the most faithful way for us to integrate into Rails.

Agreed. I think you once mentioned how performing 'hidden changes' :vampire:  to their application is a bit of an 🚩anti-pattern 🚩.

I'll make the change!